diff -urN aldo-0.6.3-orig/libdatafile/Makefile aldo-0.6.3/libdatafile/Makefile
--- aldo-0.6.3-orig/libdatafile/Makefile	2003-04-09 11:30:24.000000000 +0200
+++ aldo-0.6.3/libdatafile/Makefile	2004-01-30 16:59:11.000000000 +0100
@@ -20,9 +20,9 @@
 	@rm -f $(OBJECTS) $(PACKAGE)
 
 install: all
-	rm -Rf /usr/include/datafile
-	mkdir /usr/include/datafile
-	cp $(HEADERS) /usr/include/datafile
-	cp lib/$(NAMELIB) /usr/lib/libdatafile.so
-	ldconfig
+	rm -Rf $(DESTDIR)/usr/include/datafile
+	mkdir $(DESTDIR)/usr/include/datafile
+	cp $(HEADERS) $(DESTDIR)/usr/include/datafile
+	cp lib/$(NAMELIB) $(DESTDIR)/usr/lib/libdatafile.so
+#	ldconfig
 	@echo type 'make clean'
diff -urN aldo-0.6.3-orig/Makefile aldo-0.6.3/Makefile
--- aldo-0.6.3-orig/Makefile	2003-11-20 21:56:49.000000000 +0100
+++ aldo-0.6.3/Makefile	2004-01-30 16:57:03.000000000 +0100
@@ -63,9 +63,6 @@
 clean_aldo:
 	@$(RM) aldo $(OBJECTS)
 
-install: libs aldo uninstall
+install: libs aldo
 	@strip aldo
-	@install -m555 -oroot aldo /usr/bin/
-
-uninstall:
-	@$(RM) /usr/bin/aldo
+	@install aldo $(DESTDIR)/usr/bin/
